lessons/intro/testing/index.md

```diff
@@ -732,66 +732,67 @@ Na to uživatel spustí pytest samotný.
 
 Další informace jsou v [dokumentaci pytestu](http://doc.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner).
 
-Travis CI
----------
+GitHub Actions
+--------------
 
 Vaše testy nemusí běžet jen u vás na počítači, ale můžete je pouštět automaticky
-na službě Travis CI při každém pushnutí na GitHub.
-
-Travis CI je zadarmo pro veřejné repozitáře na [travis-ci.org], pro soukromé
-repozitáře je placená verze na [travis-ci.com]. V rámci studentského balíčku
-můžete i tuto verzi využít zdarma.
-
-Přihlaste se na [travis-ci.com] pomocí GitHubu (vpravo nahoře).
-Pak opět vpravo nahoře zvolte [Accounts](https://travis-ci.com/profile)
-a povolte Travis pro váš repozitář.
+pomocí služby [GitHub Actions] (případně jiného CI).
 
-Do repozitáře přidejte soubor `.travis.yml`:
+[GitHub Actions] je služba pro běh workflow spouštěných různými událostmi na GitHubu.
+Tato služba je zadarmo pro veřejné repozitáře (pro privátní je zdarma jen omezený 
+počet minut běhu měsíčně). Pod workflow si můžete představit skript složený z více 
+kroků, které se za sebou spouští na vzdáleném (virtuálním) stroji.
+ 
+Přesvědčte se, že v *Settings > Actions* vašeho repozitáře jsou *Actions* povolené.
+Do repozitáře přidejte soubor `.github/workflows/main.yml`:
 
 ```yaml
-language: python
-python:
-- '3.6'
-install:
-- python setup.py install
-script:
-- python setup.py test
-```
-
-Uvedený příklad je pro Python 3.6.
-Pro Python 3.7 je třeba nastavit novější verzi Ubuntu:
-
-```yaml
-language: python
-python:
-- '3.7'
-dist: xenial
-install:
-- python setup.py install
-script:
-- python setup.py test
-```
-
-Verze Pythonu lze kombinovat:
+name: Test my package
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install package
+      run:  python setup.py install
+    - name: Run tests
+      run:  python setup.py test
+```
+
+Verze Pythonu lze kombinovat pomocí tzv. *matrix*:
 
 ```yaml
-language: python
-python:
-- '3.6'
-- '3.7'
-dist: xenial
-install:
-- python setup.py install
-script:
-- python setup.py test
-```
-
-Po pushnutí by se na Travisu měl automaticky spustit test.
+name: Test my package
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
The source file is Jinja and Travis fails with `jinja2.exceptions.UndefinedError: 'matrix' is undefined`. This can be either:

 - jinja-escaped
 - or turned off in course matadata, but I fail to recall how

Reply:
OK, so this is how:


    {% raw %}
    ```yaml
    ...
    ```
    {% endraw %}

Reply:
> turned off in course matadata, but I fail to recall how

It's `jinja: False` in the lesson's `info.yml` – but it'll turn off `{{ lesson_url(...) }}` as well.
IMO `{% raw %}` is a better choice.

Reply:
Thank You, @encukou.